src/devices/machine/smc91c9x.cpp

```diff
@@ -140,14 +137,42 @@ void smc91c9x_device::device_start()
 
 void smc91c9x_device::device_reset()
 {
+	unsigned char i;
+	const unsigned char * mac;
+
 	memset(m_reg, 0, sizeof(m_reg));
+
+	memset(m_rx, 0, sizeof(m_rx));
+	memset(m_tx, 0, sizeof(m_tx));
+
 	memset(m_regmask, 0, sizeof(m_regmask));
 	m_irq_state = 0;
 	m_alloc_count = 0;
-	m_fifo_count = 0;
+	rx_fifo_out = 0;
+	rx_fifo_in = 0;
+
+	tx_fifo_out = 0;
+	tx_fifo_in = 0;
+
 	m_sent = 0;
 	m_recd = 0;
 
+	osd_list_network_adapters();
+
+	mac = (const unsigned char *)get_mac();
+
+	if (LOG_ETHERNET)
+	{
+		logerror("MAC : ");
+		for(i=0;i<ETHERNET_ADDR_SIZE;i++)
```

Comment:
This is inconsistent with the spacing above - you've got a space between the control keyword and opening parenthesis of the expression with `if (LOG_ETHERNET)` but no space between the control keyword and the parenthesis with `for(i=0;...`

Also, jamming all the tokens together without spaces like this doesn't promote readability and is inconsistent with the assignments above where there are spaces around the operators.

Reply:
done in the new PR.